html/semantics/the-button-element/command-and-commandfor/generic-eventtarget-crash.html

```diff
@@ -2,17 +2,17 @@
 <meta charset="utf-8" />
 <meta name="author" title="Keith Cirkel" href="mailto:wpt@keithcirkel.co.uk" />
 <link rel="help" href="https://open-ui.org/components/invokers.explainer/" />
-<div id="invoker"></div>
+<div id="invokee"></div>
```

Comment:
Why not just use something like "target"? It seems that for native speakers it's already easy to confuse "er" and "ee". Let alone non-native speakers.